recorder/record_subscription.py

```diff
@@ -58,7 +62,7 @@ def run(self):
                 json_data = self.sub.json
                 json_data["time"] = granted_time
                 measurement = MeasurementArray(**self.sub.json)
-
+                print(measurement)
```

Comment:
Make this a log statement?

Reply:
removed.  intend to use web sockets here later on